router/batchrouter/asyncdestinationmanager/bing-ads/offline-conversions/bulk_uploader.go

```diff
@@ -0,0 +1,344 @@
+package bingads
```

Comment:
package names can lead to confusion in both audience and offline conversions, they are same but folder path is different

Reply:
done